### PR TITLE
Computing editableValues

### PR DESCRIPTION
**Pull request by andresfulla**
@darquesdev

### CODE REVIEW

**Comment by andresfulla** [comment]
@p2kmgcl can you take a look to the last commit a please?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/content/MapContentForm.es.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
 	EDITABLE_FRAGMENT_ENTRY_PROCESSOR,
 	FRAGMENTS_EDITOR_ITEM_TYPES
 } from '../../utils/constants';
+import {computeEditableValue} from '../../utils/segmentedValues.es';
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/content/MapContentForm.es.js]: `segmentedValues` is not the name of the year

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/utils/segmentedValues.es.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+			result.value = isNullOrUndefined(preferedTranslatedValue)
+				? editableValue[defaultLanguageId]
+				: editableValue[selectedExperienceId];
+		}
+	}
+
+	return result;
+}
+
+export function computeConfigurationEditableValue(
```
Comment (andresfulla) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/utils/segmentedValues.es.js]: - [ ] Docu

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js [hunk 3]
```diff
@@ -54,7 +54,7 @@ import {
 	FREEMARKER_FRAGMENT_ENTRY_PROCESSOR
 } from '../../utils/constants';
 import {isDropZone} from '../../utils/isDropZone.es';
-import {prefixSegmentsExperienceId} from '../../utils/prefixSegmentsExperienceId.es';
+import {computeConfigurationEditableValuementedConfigurationValues} from '../../utils/segmentedValues.es';
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js]: what?

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js [hunk 4]
```diff
@@ -161,10 +154,10 @@ class FragmentEntryLink extends Component {
 			].defaultConfigurationValues;
 
 			if (configurationValues) {
-				const segmentedConfigurationValues =
-					configurationValues[segmentsExperienceId] ||
-					configurationValues[defaultSegmentsExperienceId] ||
-					configurationValues;
+				const segmentedConfigurationValues = computeConfigurationEditableValuementedConfigurationValues(
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js]: image

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js (hunk 3 above)
Comment (andresfulla) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/js/components/fragment_entry_link/FragmentEntryLink.es.js]: that might not work as intended

**Pull request closed without merging**